src/state/actions/index.js

```diff
@@ -3,5 +3,13 @@ import * as contactPreferences from './contactPreferencesActions';
 import * as referral from './referralsActions';
 import * as document from './documentUploadActions';
 import * as setAuthToken from './authTokenAction';
+import * as setProfile from './profileAction';
 
-export { client, contactPreferences, referral, document, setAuthToken };
+export {
```

Comment:
Love the organization of this code! Reads much cleaner then when it's in one line.